.travis.yml

```diff
@@ -65,6 +68,11 @@ x-definitions:
       # travis doesn't install yarn automatically on osx
       - if [[ $TRAVIS_OS_NAME == "osx" ]]; then brew install yarn --without-node; fi
 
+      # instally python3 on osx builds
```

Comment:
“Instally” all the things

Reply:
my most common typo is writing "type" instead of "typo"

Reply:
🙃